drivers/platform/xilinx/xilinx_delay.c

```diff
@@ -42,6 +42,7 @@
 /******************************************************************************/
 
 #include "no_os_delay.h"
+#include "no_os_util.h"
```

Comment:
You're adding a new file, which might break some projects which don't include this file in src.mk.

Let's wait for merging #1599 which fixes unrelated build errors and then we'll see if this commit breaks anything.